### DISCUSSION TITLE
How to disable viewing IP Address of Collaborators?

### DISCUSSION THREAD

**Discussion by SatlyBPro** [community/community#123172] [Repositories] [answered]
When I sent an invite to my friend, it showed them this when they tried to accept it, how do I stop it from wanting to see their IP Address? I don't want to view it.

![image](https://github.com/community/community/assets/144594804/c36ff787-8605-45c9-bcfc-53ed66f5ce7b)

**Comment by techpond31** [comment, marked as answer]
🤔 Based on the screenshot you shared, it appears your friend received an email invitation asking him to join a Team on GitHub. During the account linking process, GitHub prompts the user for consent to share certain details with third-party applications, one of them being the IP address. Since GitHub requires authentication to manage teams programmatically, it collects IP addresses to ensure legitimate access to sensitive operations.

As a host or creator of the team, you won't be able to control whether GitHub shows the IP address collection prompt to invited users. However, keep in mind that neither you nor anyone else from the team will ever receive the IP address of the person accepting the invitation. That data remains confidential and only collected for internal purposes to protect GitHub services.

Additionally, GitHub complies with applicable laws and regulations concerning data protection and privacy, ensuring transparency around how personal data gets handled. More information about this topic can be found in GitHub's Privacy Statement.

While you cannot remove the prompt entirely, assure your friend that no one involved in the team creation process can view his IP address. By continuing past this screen, he grants temporary permissions allowing GitHub to authenticate the acceptance process accurately. Once accepted, the IP address-related information ceases to play a role in further interactions with the platform.

Encourage your friend to click "Continue" in order to complete the account linking process. Rest assured that sharing IP addresses occurs strictly adhering to GitHub's strict privacy rules and principles.
